marathon_lb.py

```diff
@@ -805,7 +808,7 @@ def config(apps, groups, bind_http_https, ssl_certs, templater):
             healthCheckOptions = None
             if app.healthCheck:
                 server_health_check_options = None
-                if app.mode == 'tcp':
+                if app.mode == 'tcp' or app.healthCheck['protocol'] == 'TCP':
```

Comment:
I think you're right...although it's confusing that Marathon uses lowercase for the portmappings, but uppercase for the health checks.